app/build.gradle.kts

```diff
@@ -22,6 +22,7 @@ dependencies {
     implementation(project.dependencies.platform(libs.koin.bom))
     implementation(libs.koin.core)
     implementation(libs.koin.android)
+    implementation(libs.koin.compose)
```

Comment:
❓ I would say we are not using anything from that library. We may, but not for the moment, right?